ext/libxml/tests/bug76777.phpt

```diff
@@ -0,0 +1,41 @@
+--TEST--
+Bug #76777 (first parameter of libxml_set_external_entity_loader callback undefined)
+--SKIPIF--
+<?php if (!extension_loaded('libxml')) die('skip'); ?>
+--FILE--
+<?php
+ini_set('error_reporting',PHP_INT_MAX-1);
+$xml=<<<EOF
+<?xml version="1.0"?>
+<test/>
+EOF;
+
+$xsd=<<<EOF
+<?xml version="1.0"?>
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema">
+  <xs:include schemaLocation="nonexistent.xsd"/>
+    <xs:element name="test"/>
+    </xs:schema>
+EOF;
+
+libxml_set_external_entity_loader(function($p,$s,$c) {
+    var_dump($p,$s,$c);
+        die();
```

Comment:
To `die` here seems doubtful. I'd rather remove the `var_dump` 4 lines below.

Reply:
`die` is there to avoid schema parsing errors on output as they are not relevant to the test, but are artifact of simplistic test case:

>Warning: DOMDocument::schemaValidateSource(): Element '{http://www.w3.org/2001/XMLSchema}include': Failed to parse the XML resource 'nonexistent.xsd'. in /in/Tt5TM on line 20
>
>Warning: DOMDocument::schemaValidateSource(): Invalid Schema in /in/Tt5TM on line 20

Reply:
Ah, okay. Thanks!